config/webpack.client.prod.js

```diff
@@ -25,6 +25,7 @@ const config = Object.assign({}, defaults, {
   }),
   plugins: defaults.plugins.concat([
     new ManifestPlugin({ fileName: 'manifest.json' }),
+    new ManifestPlugin({ fileName: `manifest.${Date.now()}.json` }),
```

Comment:
Yeah, manifest.json will still be there. This creates two copies of the manifest so you get both `manifest.json` and `manifest.timestamp.json` this is useful for figuring out which files were related to what builds over time, even though the original `manifest.json` will have been overwritten.